core/app/models/workarea/audit_logging.rb

```diff
@@ -0,0 +1,32 @@
+module Workarea
+  module AuditLogging
+    extend ActiveSupport::Concern
+    include Mongoid::AuditLog
+
+    class_methods do
+      def enable_audit_log
+        @audit_log_enabled = true
```

Comment:
Should these be thread scoped?

Reply:
no, looking to disable permanently.

Reply:
We could probably do something a bit more useful in the gem itself with allowing temporary disabling/enabling as an option